src/governance-vote.h

```diff
@@ -19,31 +19,76 @@ using namespace std;
 
 class CGovernanceVote;
 
-#define VOTE_OUTCOME_NONE     0
-#define VOTE_OUTCOME_YES      1
-#define VOTE_OUTCOME_NO       2
-#define VOTE_OUTCOME_ABSTAIN  3
-// INTENTION OF MASTERNODES REGARDING ITEM
-
-#define VOTE_SIGNAL_NONE                0 // SIGNAL VARIOUS THINGS TO HAPPEN:
-#define VOTE_SIGNAL_FUNDING             1 //   -- fund this object for it's stated amount
-#define VOTE_SIGNAL_VALID               2 //   -- this object checks out in sentinel engine
-#define VOTE_SIGNAL_DELETE              3 //   -- this object should be deleted from memory entirely
-#define VOTE_SIGNAL_ENDORSED            4 //   -- officially endorsed by the network somehow (delegation)
-#define VOTE_SIGNAL_NOOP1               5 // FOR FURTHER EXPANSION
-#define VOTE_SIGNAL_NOOP2               6 // 
-#define VOTE_SIGNAL_NOOP3               7 // 
-#define VOTE_SIGNAL_NOOP4               8 // 
-#define VOTE_SIGNAL_NOOP5               9 // 
-#define VOTE_SIGNAL_NOOP6               10 // 
-#define VOTE_SIGNAL_NOOP7               11 // 
-#define VOTE_SIGNAL_NOOP8               12 // 
-#define VOTE_SIGNAL_NOOP9               13 // 
-#define VOTE_SIGNAL_NOOP10              14 // 
-#define VOTE_SIGNAL_NOOP11              15 // 
-#define VOTE_SIGNAL_CUSTOM_START        16 // SENTINEL CUSTOM ACTIONS 
-#define VOTE_SIGNAL_CUSTOM_END          35 //        16-35
+enum vote_outcome_enum_t  {
+    VOTE_OUTCOME_NONE      = 0,
+    VOTE_OUTCOME_YES       = 1,
+    VOTE_OUTCOME_NO        = 2,
+    VOTE_OUTCOME_ABSTAIN   = 3
+};
 
+// #define VOTE_OUTCOME_NONE     0
```

Comment:
Should probably just move valuable comments down to new enum values and remove old `#define`s completely instead of commenting them out.